buildConfigs.js

```diff
@@ -7,11 +7,11 @@ const sharedConfig = {
   entryPoints: ["src/index.tsx", "src/useChat.ts"],
   logLevel: "info",
   treeShaking: true,
-  minify: true,
-  sourcemap: true,
+  minify: false,
+  sourcemap: false,
   external: [...Object.keys(dependencies), ...Object.keys(devDependencies), ...Object.keys(peerDependencies)],
   target: ["esnext", "node12.22.0"],
-  plugins: [cssPlugin(), sassPlugin({ type: "style" })],
+  plugins: [cssPlugin(), sassPlugin({ type: "css-text" })],
```

Comment:
Can you help me understand the changes to this file? Why is minify and sourcemap set to false, and why is the sassPlugin type changed? Maybe some code comments would be helpful.

Reply:
I'll add comments, but since you asked here:
1. Thanks for catching minify: false. I meant to restore that to true.
2. Removing the source map theoretically saves us some 100kb in package size. I re-enable that when building the package locally via the `npm run docs` command since that doubles as our development environment.
3. esbuild's sass plugin exposes multiple output types. `style` basically embeds css content in style tags that get appended to the DOM, which makes it perfect for standalone apps. `css-text` outputs style as text strings, and are recommended for use with Web Components [in the plugin docs](Point to different output directories).

Reply:
https://github.com/vectara/react-chatbot/pull/9/commits/d439e717267622629e3ea5c4983c52e9dfa59d51